kubectl-cloudflow/cmd/deploy.go

```diff
@@ -132,6 +123,36 @@ You can update the credentials with the "update-docker-credentials" command.
 	rootCmd.AddCommand(deployOpts.cmd)
 }
 
+// validateVerifyCmdArgs validates the args for the verify command.
+// we only accept one argument at the moment which is a path to the blueprint.
+func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
+	if cmd.Flag("image").Changed == false && cmd.Flag("blueprint").Changed == false {
+		return fmt.Errorf("Please specify either the blueprint file or the image path")
+	}
+	if (cmd.Flag("image").Changed == true) && (cmd.Flag("blueprint").Changed == true) {
+		return fmt.Errorf("%s", "Cannot specify both blueprint and image in command")
+	}
+	if cmd.Flag("image").Changed == true {
+		util.PrintSuccess("Deploying image %s\n", cmd.Flag("image").Value.String())
+	} else if cmd.Flag("blueprint").Changed == true {
+
+		blueprintFile := cmd.Flag("blueprint").Value.String()
+		blueprintURL, err := url.Parse(blueprintFile)
```

Comment:
does this mean we support remote urls. In `verify` I kept this functionality should I remove it?

Reply:
@skonto I copied it from `verify` :-)

Reply:
@debasishg sure :) My question is do we support ```kubectl cloudflow deploy --blueprint https://...../call-record-aggregator-blueprint.conf```?

Reply:
@skonto I am not sure we need this. But even if we decide to change we can do it in a separate PR. Maybe @RayRoestenburg can pitch in as well with his thoughts ?

Reply:
I think in deploy you are supporting it already as you use `fileutils.GetFileContents` which supports remote urls.